vispy/gloo/glir.py

```diff
@@ -844,8 +844,7 @@ def _gl_initialize(self):
             else:
                 this_version = this_version[0]
 
-            this_version = LooseVersion(this_version)
-            if this_version < '2.1':
+            if this_version and LooseVersion(this_version) < '2.1':
```

Comment:
I ran into `this_version` being an empty string, causing `LooseVersion` to fail.

Reply:
This is usually a sign that OpenGL isn't being loaded correctly or isn't being found. At least in my experience. It has been on the radar for a long time to improve the error message that is produced, but if we silently let it through then it may confuse some users. In what cases is the GL version being an empty string not a sign of something being wrong?